csvcubed/csvcubed/cli/build.py

```diff
@@ -1,13 +1,24 @@
 """
 Build Command
 -------------
-Build a qb-flavoured CSV-W from an info.json and a tidy CSV.
+Build a qb-flavoured CSV-W from a config.json and a tidy CSV.
 """
+import dataclasses
+import json
 import logging
 from pathlib import Path
 from typing import Optional
 
+from csvcubedmodels.dataclassbase import DataClassBase
+
+from csvcubed.readers.configdeserialiser import (
+    get_cube_from_data,
+    get_cube_from_config_json
+)
+from csvcubed.utils.qb.validation.cube import validate_qb_component_constraints
+
 _logger = logging.getLogger(__name__)
+_logger.addHandler(logging.StreamHandler())
```

Comment:
We don't want to merge this one in.

Isn't the logger output configured in Muazzam's code? I know there were some bugs in it that I've fixed. If you merge a fresh copy of `origin/main` in it should make sure it works.

Reply:
removed